src/ToricVarieties/ToricDivisors/properties.jl

```diff
@@ -67,24 +67,55 @@ export is_basepoint_free
 @doc Markdown.doc"""
     is_effective(td::ToricDivisor)
 
-Determine whether the toric divisor `td` is effective.
+Determine whether the toric divisor `td` is effective,
+i.e. if all of its coefficients are non-negative.
 
 # Examples
 ```jldoctest
-julia> F4 = hirzebruch_surface(4)
-A normal, non-affine, smooth, projective, gorenstein, non-fano, 2-dimensional toric variety without torusfactor
+julia> P2 = projective_space(NormalToricVariety,2)
+A normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
 
-julia> td = ToricDivisor(F4, [1,0,0,0])
-A torus-invariant, prime divisor on a normal toric variety
+julia> td = ToricDivisor(P2, [1,-1,0])
+A torus-invariant, non-prime divisor on a normal toric variety
 
 julia> is_effective(td)
+false
+
+julia> td2 = ToricDivisor(P2, [1,2,3])
+A torus-invariant, non-prime divisor on a normal toric variety
+
+julia> is_effective(td2)
 true
 ```
 """
-@attr Bool is_effective(td::ToricDivisor) = pm_tdivisor(td).EFFECTIVE
+@attr Bool is_effective(td::ToricDivisor) = all(c -> (c >= 0), coefficients(td))
 export is_effective
 
 
+@doc Markdown.doc"""
+    is_linearly_equivalent_to_effective_toric_divisor(td::ToricDivisor)
+
+Determine whether the toric divisor `td` is linearly equivalent to an effective toric divisor.
+
+# Examples
+```jldoctest
+julia> P2 = projective_space(NormalToricVariety,2)
+A normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> td = ToricDivisor(P2, [1,-1,0])
+A torus-invariant, non-prime divisor on a normal toric variety
+
+julia> is_effective(td)
+false
+
+julia> is_linearly_equivalent_to_effective_toric_divisor(td)
+true
+```
+"""
+@attr Bool is_linearly_equivalent_to_effective_toric_divisor(td::ToricDivisor) = pm_tdivisor(td).EFFECTIVE
+export is_linearly_equivalent_to_effective_toric_divisor
```

Comment:
Do we really want this? Can't one just use the class construction? I mean we could end up with tons of `is_linearly_equivalent_to_*` functions. Is this something we want to avoid or would it be ok?

Reply:
I thought this might be convenient, but on second thought you might be right. As I see it (now), we could remove this.